src/Components/Components/src/Routing/Router.cs

```diff
@@ -23,7 +23,7 @@ public class Router : IComponent, IHandleAfterRender, IDisposable
         static readonly ReadOnlyDictionary<string, object> _emptyParametersDictionary
             = new ReadOnlyDictionary<string, object>(new Dictionary<string, object>());
 
-        RenderHandle _renderHandle;
+        internal RenderHandle _renderHandle { get; set; }
```

Comment:
```suggestion
        internal RenderHandle RenderHandle { get; set; }
```
Naming conventions when making this a property.

Reply:
Good call! I'll hold off on accepting the suggestion since it requires changing invocations in the Router class. Also, I'd like to see if there is a way to avoid having to make this change at all.